src/server/routes/index.js

```diff
@@ -0,0 +1,37 @@
+const db = require('../../models/db/db');
+const todos = require('../../models/db/todos');
+const users = require('../../models/db/users');
+
+const router = require('express').Router();
+
+router.get('/', (request, response) => {
+  response.render('todo/todo-list');
+});
+
+router.get('/signup', (request, response) => {
+  response.render('users/sign-up');
+});
+
+router.post('/signup', (request, response) => {
+  users.signUp(request.body)
+  .then((user) => {
```

Comment:
indent this `then` block

Reply:
also should you do some check here? if `user` is true would go to profile page? What happens if user didn't get signed up here?

Reply:
Complete 👍
And the fields are required. If the user does not signup then nothing happens. They would need to sign up. What I can do is add a link to the form to send the user to the signin page if they need to signin instead. When I user logs in, they will immediately see the todo list form and their todos.